src/nools/lib.js

```diff
@@ -125,7 +140,9 @@ function emitTasksForSchedule(c, schedule, r) {
         if(event.dueDate) {
           dueDate = event.dueDate(event, c);
         } else {
-          dueDate = new Date(Utils.addDate(new Date(c.contact.reported_date), event.days));
+          // The default is the day the user was created?? That makes no sense.
```

Comment:
I think `c.contact` is the patient here, not the user. 
This is how, I think, `c` is generated: https://github.com/medic/medic/blob/master/webapp/src/js/services/rules-engine.js#L61 . 
In this context, it makes sense to default the `dueDate` to the moment the contact was created. We already have this behavior in `standard` config with SMS (https://github.com/medic/medic/blob/master/config/standard/app_settings.json#L587) where for a newly created person, we schedule an SMS 7 days after creation.

Reply:
Oh! ... when you have a report about a contact you don't have, `c.contact` ends up being the current user?

Reply:
Probably shouldn't keep that comment in there :) I think you're right it is in accurate. `c.contact.reported_date` is the timestamp when the contact document was created. I agree it is a sensible thing to use sometimes, but still a bit of a strange default. It's hard just because there isn't really a clear dueDate for contacts.

Before 3.6, if you have a "headless" report then `c.contact` is undefined. After the proposed change in >3.6, `c.contact` is a skeleton object containing only the `id`.